src/main/java/appeng/me/storage/MEInventoryHandler.java

```diff
@@ -129,7 +129,9 @@ public IItemList<T> getAvailableItems(final IItemList<T> out, int iteration) {
     }
 
     public boolean isVisible() {
-        return this.internal instanceof MEMonitorIInventory inv && inv.getMode() == StorageFilter.NONE;
+        boolean bool = this.internal instanceof MEMonitorIInventory inv && inv.getMode() == StorageFilter.NONE;
+        if (this.internal instanceof MEMonitorPassThrough inv && inv.getMode() == StorageFilter.NONE) bool = true;
+        return bool;
```

Comment:
Rather than carrying a variable named "bool", why not the following of just checking either then returning true?
```suggestion
        if (this.internal instance of MEMonitorIInventory inv || this.internal instanceof MEMonitorPassThrough inv) {
            if (this.getMode() == StorageFilter.NONE) return true;
        }
        return false;
```